### DISCUSSION TITLE
Punkteberechnung

### DISCUSSION THREAD

**Discussion by obermacker** [obermacker/spacefights#10] [General]
Hallo, ich habe eine Frage zu dieser Zeile:

[https://github.com/obermacker/spacefights/blob/0a1dd81d24be48bf690bd5df9e6a1d25448e4766/galaxy1/inc/func_galaxy.php#L815](https://github.com/obermacker/spacefights/blob/0a1dd81d24be48bf690bd5df9e6a1d25448e4766/galaxy1/inc/func_galaxy.php#L815)

```
// !!! punkte update macht so keinen sinn !!!
//$punkte = get_punkte($player_id, $planet_id);
//$punkte = $punkte + ((($Deff["Kosten_Eisen"] + $Deff["Kosten_Silizium"] + $Deff["Kosten_Wasser"]) * $anzahl) / 1000);
//
//$abfrage_planet_update = "UPDATE `planet` SET  `$tabelle` = $deff_soll
//, `punkte` = " . "0" . " WHERE `Spieler_ID` = '$player_id' AND `Planet_ID` = $planet_id";
//$query = $abfrage_planet_update or die("Error in the consult.." . mysqli_error("Error: set_bauschleife_struckture #1 ".$link));
```

Was macht daran keinen Sinn? Dadurch das es Ersatzlos auskommentiert wurde, berechnet er ja an der Stelle gar keine Punkte mehr.

VG Steve

**Comment by schimmi70** [comment]
Ich habe das wahrscheinlich gefunden  und da es auskommentiert war und ich nicht wusste ob die Berechnung dort durchgeführt werden soll oder nicht , habe ich den Kommentar geschrieben (in meiner gewohnt höfflichen Art )  😞 ... 

**Ich wollte eher schreiben, wenn es noch gebraucht wird aktivieren / ansonsten löschen ?**

**Comment by obermacker** [comment]
Ich fand es als Spieler immer großartig, wenn man seine Punkte direkt gutgeschrieben bekam.

Der Code gehört aber tatsächlich in eine Funktion.
